src-java/flowhs-topology/flowhs-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/bolts/HaFlowCreateHubBolt.java

```diff
@@ -173,9 +175,11 @@ public void sendNotifyFlowMonitor(@NonNull CommandData flowCommand) {
 
     @Override
     public void sendPeriodicPingNotification(String haFlowId, boolean enabled) {
-        //TODO implement periodic pings
-        log.info("Periodic pings are not implemented for ha-flow create operation yet. Skipping for the ha-flow {}",
-                haFlowId);
+        log.debug("Periodic ping ha-flow create operation. haFlowId={}", haFlowId);
+        PeriodicPingCommand payload = new PeriodicPingCommand(haFlowId, enabled);
+        Message message = new CommandMessage(payload, getCommandContext().getCreateTime(),
+                getCommandContext().getCorrelationId());
+        emitWithContext(Stream.HUB_TO_PING_SENDER.name(), getCurrentTuple(), new Values(currentKey, message));
```

Comment:
Please do the same for HaFlowDeleteHubBolt

Reply:
done

Reply:
Looks like your branch is outdated. I see `HaFlowUpdateHubBolt` with empty `sendNotifyFlowMonitor`, 
But I don't see `HaFlowPathSwapHubBolt` and `HaFlowRerouteHubBolt` like in your base branch

Reply:
I just rebased it and implemented the missing methods.

Reply:
but looks like you missed HaFlowRerouteHubBolt